src/pudl_archiver/archivers/classes.py

```diff
@@ -58,8 +58,12 @@ class AbstractDatasetArchiver(ABC):
     directory_per_resource_chunk: bool = False
 
     # Configure which generic validation tests to run
-    check_missing_files: bool = True
-    check_empty_invalid_files: bool = True
+    fail_on_missing_files: bool = True
```

Comment:
non-blocking: In the future, if we add more validation tests, we might want to refactor this so that the `fail_on_x` and the `required_for_run_success` flags aren't duplicated - but that's not really an issue at this juncture IMO.

Reply:
The `required_for_run_success` flags are taking the `fail_on_x` parameters as boolean inputs at this point, so I'm not 100% following what would be the proposed change here?

Reply:
Right now we have to make sure to pass the `fail_on_x` parameters in to the `required_for_run_success` flags - which is easy enough, but still error-prone. We *should* be able to rejigger the class structure so that we can't accidentally break it, but that definitely seems like too big of a refactor for this PR.